.github/CODEOWNERS

```diff
@@ -1,7 +1,6 @@
-# CODEOWNERS file for BuxOrg
+# CODEOWNERS file for Bitcoin SV
 
```

Comment:
Remove those lines like in the other repositories

Reply:
Done